sycl/test-e2e/OptionalKernelFeatures/no-fp64-optimization-declared-aspects.cpp

```diff
@@ -0,0 +1,35 @@
+// UNSUPPORTED: aspect-fp64
+// RUN: %{build} -o %t.out -O3
```

Comment:
Is there any way to test described "compiler actions" apparently? For example, perform these actions manually.
Currently, it looks like we test compiler's "magic" behavior.

Reply:
Such tests should be present under `clang/Driver` in general. However, I don't quite understand why would we need such a test in this particular scenario: we haven't changed compilation flow at all, we made modifications to a single pass, so that it produces slightly altered output. The changes are tested by (a) isolated LIT test, which ensures that the pass (or rather `sycl-post-link` as a whole) produces expected output and (b) end-to-end test to ensure that complete example we have problems with  works fine

Reply:
Didn't notice LIT test. Probably, it is enough.